nototools/unicode_data.py

```diff
@@ -32,7 +32,11 @@
 import os
 from os import path
 import re
-import sys
+
+try:
+  basestring
+except NameError:
+  basestring = str
```

Comment:
pls import basestring from nototools.py23

Reply:
Should be all good now.